### DISCUSSION TITLE
Not found Plots (one key, one PC)

### DISCUSSION THREAD

**Discussion by dro737** [Chia-Network/chia-blockchain#5878] [Support]
Hello,

I'm having issue with Chia GUI not finding my plots, I have 36 in total which it only finds 6 out of 36, tried resetting the plot directory and refreshing with still same issue, all plots were created from one pc using one key. all 36 plots are on external HDD but somehow GUI only finding 6 out of 36.

in the Not found Plots section, I copied the file name and searched for plot and can find them immediately, so not sure why it can't find the other 30, the path is still the same. 

I tried moving one plot to an internal HDD to see if it will get discovered but it won't and I don't feel like deleting 30 plots and remaking them.

Tried uninstalling and reinstalling which doesn't help, have spent last 4 days syncing and now after that I am missing the plots and can't properly farm everything. quite frustrating really which makes me want to give up on this network

would like some help/guide on this please

**Comment by segovia888** [comment]
Hola, estoy como tu también me han desaparecido 33 parcelas, en el disco duro están pero no me las detecta

**Comment by BrandtH22** [comment]
@dro737 , if you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
